terraform/cloudfoundry/network_acls_cf_in.tf

```diff
@@ -172,6 +139,15 @@ resource "aws_network_acl_rule" "110_bosh" {
     egress = false
 }
 
+resource "aws_network_acl_rule" "115_router_in" {
+    network_acl_id = "${aws_network_acl.cf.id}"
+    protocol = -1
+    rule_number = 115
+    rule_action = "allow"
+    cidr_block = "${var.router_cidr_all}"
```

Comment:
Can we be more specific about the ports allowed here? Do we know what the router needs to talk to?